readthedocs-theme/templates/includes/footer.html

```diff
@@ -0,0 +1,170 @@
+              <p>Sign up for our newsletter to get our latest blog updates delivered to your inbox weekly.</p>
+
+              <div class="field">
+                <label for="mce-EMAIL">Email</label>
+                <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="you@example.com">
+              </div>
+
+              <div id="mce-responses">
+                <div class="ui hidden response" id="mce-error-response"></div>
+                <div class="ui hidden response" id="mce-success-response"></div>
+              </div>
+
+              <input class="ui primary button" type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button">
+            </form>
+          </div>
+        </div>
+
+        <div class="column">
+          <h4 class="ui sub header">Get Involved</h4>
+
+          <div class="ui list">
+            <a class="item" href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html">Getting Started Guide</a>
+            <a class="item" href="https://docs.readthedocs.io">Documentation</a>
+            <a class="item" href="https://docs.readthedocs.io/page/contribute.html">Contributing</a>
+            <a class="item" href="https://docs.readthedocs.io/page/team.html">Team</a>
+            <a class="item" href="https://docs.readthedocs.io/page/open-source-philosophy.html">Open Source Philosophy</a>
+          </div>
+        </div>
+
+        <div class="column">
+          <h4 class="ui sub header">Business Info</h4>
+
+          <div class="ui list">
+            <a class="item" href="">Advertise with Us</a>
+            <a class="item" href="https://readthedocs.com">Read the Docs for Business</a>
+            <a class="item" href="">Supporters</a>
+            <a class="item" href="https://docs.readthedocs.io/page/privacy-policy.html">Privacy Policy</a>
+            <a class="item" href="https://docs.readthedocs.io/page/terms-of-service.html">Terms of Service</a>
+            <a class="item" href="https://read-the-docs-guidelines.readthedocs-hosted.com/">Branding</a>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="ui basic center aligned segment">
+        <div class="ui very relaxed horizontal list">
+          <div class="left aligned item">
+            <i class="large copyright basic middle aligned icon"></i>
+            <div class="content">
+              <div class="header">Copyright</div>
+              <div class="description">2021, Read the Docs, Inc & contributors</div>
+            </div>
+          </div>
+          <div class="left aligned item">
+            <i class="large code branch icon"></i>
+            <div class="content">
+              <div class="header">Version</div>
+              <div class="description">
+                <a href="http://docs.readthedocs.io/page/changelog.html">
+                  <b>6.1.0</b>
+                </a>
```

Comment:
This block will be used on the application, but perhaps this is another one to drop form our website -- where the application version doesn't make as much sense, and would get out of sync unless we update it each release.